src/main/java/cn/nukkit/network/protocol/ProtocolInfo.java

```diff
@@ -17,12 +17,12 @@
     /**
      * Actual Minecraft: PE protocol version
      */
-    int CURRENT_PROTOCOL = dynamic(448);
+    int CURRENT_PROTOCOL = Integer.valueOf("465");
```

Comment:
The protocol version need to be dynamic since otherwise plugins would need to be recompiled against the newer version.

Reply:
Improved code